public/styles/styles.css

```diff
@@ -1,10 +1,120 @@
+
+
 body {
-  color: #333;
+  color: black;
   font-family: Helvetica, Arial, sans-serif;
-  background-color: skyblue; /* Sanity Check! */
+  background-color: rgb(239, 239, 239); /* Sanity Check! */
+  margin: 0;
+
```

Comment:
You guys have been writing code long enough now where we're going to start looking at your writing style and not just the code itself. Going forward be mindful of extra spaces in these blocks (code between {}) and between blocks.